tools/unicode-check.py

```diff
@@ -10,13 +10,8 @@
 # The set of Unicode code points greater than 127 that we
 # allow in the source code.
 box_drawing_chars = set(chr(cp) for cp in range(0x2500, 0x2580))
-greek_letters = set(chr(cp) for cp in chain(
-    range(0x03b1, 0x3c2), range(0x3c3, 0x03c9 + 1),  # small
-    range(0x0391, 0x03a2), range(0x03a3, 0x03a9 + 1))  # large
-)
 allowed = (set(['®', 'é', 'ê', 'ó', 'ö', 'ü', '∫', '≠', '≥']) |
```

Comment:
Funnily enough, it seems the maintainers do add in these symbols occasionally (I see lambda in the copy in the upstream copy https://github.com/scipy/scipy/blob/5bfbe973ac849db94eb1bb42f4d51697afeed46e/tools/unicode-check.py#L13-L14), but it's good to see this change reverted for the final review.